fastparquet/writer.py

```diff
@@ -814,15 +815,27 @@ def write(filename, data, row_group_offsets=50000000,
           mkdirs=default_mkdirs, has_nulls=True, write_index=None,
           partition_on=[], fixed_text=None, append=False,
           object_encoding='infer', times='int64'):
-    """ Write Pandas DataFrame to filename as Parquet Format
+    """ Write Pandas DataFrame to filename as Parquet Format.
+
+        3 writing modes can be identified:
+            - create file(s),
+            - append data by creating new files without removing existing ones,
```

Comment:
So the first is `append=False`; the other two are `append=True` but depend on the other arguments? I would call these modes "append" and "overwrite-in-place". 

Really, it may make sense to have `append=True|False|overwrite`, and ensure that if the third is chose, `row_group_offsets==0`.

Reply:
You are right. This came to my mind as well, but as to enable it, you need a specific combination of parameters, this new value of `append` only confirms this 'overwrite-in-place' mode. Ok, doing it.